Tests/SentryTests/Protocol/SentrySDKSettings+Equality.m

```diff
@@ -0,0 +1,30 @@
+#import "SentrySDKSettings+Equality.h"
+
+@implementation SentrySDKSettings (Equality)
+
+- (BOOL)isEqual:(id _Nullable)object
+{
+    if (object == self)
+        return YES;
+    if ([self class] != [object class])
+        return NO;
+
+    SentrySDKSettings *otherSDKSettings = (SentrySDKSettings *)object;
```

Comment:
`m`: Consider using the macro for unwrapping nullable. But you will have to include "SentryInternalDefines.h".

```suggestion
    SentrySDKSettings *_Nonnull otherSDKSettings = SENTRY_UNWRAP_NULLABLE(SentrySDKSettings, object);
```